Podcast/FeedTableViewCell.swift

```diff
@@ -0,0 +1,460 @@
+//
+
+
+import UIKit
+
+class FeedTableViewCell: UITableViewCell {
+    
+    ///
+    /// Mark: View Constants
+    ///
+    var height: CGFloat = 253
+    var seperatorHeight: CGFloat = 9
+    var episodeNameLabelY: CGFloat = 27
+    var episodeNameLabelX: CGFloat = 86.5
+    var episodeNameLabelRightX: CGFloat = 21
+    var episodeNameLabelHeight: CGFloat = 18
+    var dateTimeLabelX: CGFloat = 86.5
+    var dateTimeLabelY: CGFloat = 47.5
+    var dateTimeLabelHeight: CGFloat = 14.5
+    var descriptionLabelX: CGFloat = 17.5
+    var descriptionLabelY: CGFloat = 94
+    var descriptionLabelHeight: CGFloat = 54
+    var descriptionLabelRightX: CGFloat = 11.5
+    var tagsLabelX: CGFloat = 17
+    var tagsLabelY: CGFloat = 160.5
+    var tagsLabelRightX: CGFloat = 17.5
+    var tagsLabelHeight: CGFloat = 18
+    var recommendedLabelHeight: CGFloat = 18
+    var podcastImageX: CGFloat = 17.5
+    var podcastImageY: CGFloat = 17
+    var podcastImageSize: CGFloat = 60
+    var bookmarkButtonX: CGFloat = 304
+    var bookmarkButtonHeight: CGFloat = 18
+    var bookmarkButtonWidth: CGFloat = 12
+    var recommendedButtonWidth: CGFloat = 16
+    var recommendedButtonHeight: CGFloat = 16
+    var recommendedButtonX: CGFloat = 233
+    var buttonPadding: CGFloat = 10
+    var lineSeperatorX: CGFloat = 18
+    var lineSeperatorHeight: CGFloat = 1
+    var playButtonX: CGFloat = 19.5
+    var playButtonWidth: CGFloat = 11
+    var playButtonHeight: CGFloat = 13.5
+    var moreButtonX: CGFloat = 342
+    var moreButtonHeight: CGFloat = 3
+    var moreButtonWidth: CGFloat = 15
+    var contextLabelX: CGFloat = 17
+    var contextLabelHeight: CGFloat = 30
+    var contextLabelRightX: CGFloat = 43
+    var contextImagesSize: CGFloat = 28
+    var feedControlButtonX: CGFloat = 345
+    var feedControlButtonHieght: CGFloat = 7.5
+    var feedControlButtonWidth: CGFloat = 13
+    
+    var contextViewHeight: CGFloat = 52
+    var bottomViewHeight: CGFloat = 48
+    var mainViewHeight: CGFloat = 195
+    
+    ///
+    /// Mark: Variables
+    ///
+    var heightConstraint: NSLayoutConstraint!
+    var episodeNameLabel: UILabel!
+    var dateTimeLabel: UILabel!
+    var descriptionLabel: UILabel!
+    var tagsLabel: UILabel!
+    var recommendedLabel: UILabel!
+    var recommendedButton: UIButton!
+    var bookmarkButton: UIButton!
+    var moreInfoButton: UIButton!
+    var seperator: UIView!
+    var podcastImage: UIImageView!
+    var lineSeperator: UIView!
+    var topLineSeperator: UIView!
+    var moreButton: UIButton!
+    var playButton: UIButton!
+    var playLabel: UILabel!
+    var contextLabel: UILabel!
+    var contextImages: [UIImageView] = []
+    var contextView: UIView! //view for upper context bar of feed cell
+    var mainView: UIView! //main view
+    var bottomView: UIView! //bottom bar view with buttons
+    var feedControlButton: UIButton!
+    var playing = false
+    
+    var card: Card? {
+        didSet {
+            setCard()
```

Comment:
What is the point of having this here? The object that holds this should call setCard directly instead of this hidden didset functionality